install-dev/models/install.php

```diff
@@ -356,7 +356,7 @@ public function createShop($shop_name)
         $shop->active = true;
         $shop->id_shop_group = $shop_group->id;
         $shop->id_category = 2;
-        $shop->id_theme = 1;
+        $shop->theme_name = 'StarterTheme';
```

Comment:
No need to change this now, but btw. it might be nice to *not* hard-code this and just use "the theme that's in `themes/`". This could allow ppl to package PrestaShop with a different default theme for instance.